Dockerfile

```diff
@@ -22,9 +22,21 @@ RUN apt-get update \
 	&& rm -rf /var/lib/apt/lists/* \
 	&& docker-php-ext-install curl json mysql mysqli
 
+# Use the production php.ini unless PHP_DEBUG=1 (defaults to 0)
+ARG PHP_DEBUG=0
+RUN [ "$PHP_DEBUG" = "1" ] || \
+	{ echo "Using production php.ini" \
```

Comment:
Should this be `Using development php.ini`?

Reply:
`PHP_DEBUG != 1` if for the live server, so we'll use `php.ini-production` there. `PHP_DEBUG==1` is for development, but I prefer the default values over the values in `php.ini-development`, because it too suppresses in-browser warnings and errors (they go into the logfile, which is much less convenient when debugging).

Reply:
Oh, you used `||`, I assumed it would be `&&` which is what confused me, eg `if debug is false then use prod` rather than `if debug is true ... else use prod`

Reply:
I originally had the `&&`, but the problem with that is it causes docker to exit on failure if not using production (since you get an overall false return value).

Reply:
Ah, what about `[ "$PHP_DEBUG" = "1" ] && echo 'using dev' || echo 'using prod' ...`

Reply:
Sure, that sounds good.